parent-pom.xml

```diff
@@ -78,6 +78,7 @@
     <relocationBase>net/snowflake/client/jdbc/internal</relocationBase>
     <shadeBase>net.snowflake.client.jdbc.internal</shadeBase>
     <shadeNativeBase>net_snowflake_client_jdbc_internal</shadeNativeBase>
+    <skip.ut>false</skip.ut>
```

Comment:
added parameter to be able to skip running unit tests when running authentication tests

Reply:
What does `ut` stand for? Maybe we can use full name instead?

Reply:
it's unit tests, I can change to full name